designs/sam-config.md

```diff
@@ -360,6 +360,7 @@ Documentation Changes
 
 * Addition of a new `--config-file` parameter per command
 * Addition of a new `--config-prefix` parameter per command
+* Replace `--config-prefix` by `--config-env`
```

Comment:
could we replace line 362 instead, since this looks to be a documentation miss.

Reply:
For sure np, I will remove line 363 and update line 362